flow/scheduling/simple_scheduler.py

```diff
@@ -10,20 +11,43 @@
 
 
 class SimpleScheduler(Scheduler):
+    """Implementation of the abstract Scheduler class for SimpleScheduler.
```

Comment:
This suggestion seems incorrect. This is not just a "simple" implementation, it's designed for the included SimpleScheduler. I also may have revised this docstring further in later PRs.

Reply:
I'm not sure what you mean, this _is_ the included `SimpleScheduler`. Are you referring the `SimpleSchedulerEnvironment` that is used to spoof the detection of this scheduler? Or are you concerned that just calling it a simple implementation of a scheduler doesn't adequately describe what it does?

Reply:
I mean that this is a scheduler interface for the program [`bin/simple-scheduler`](https://github.com/glotzerlab/signac-flow/blob/master/bin/simple-scheduler). That isn't "a simple implementation of the abstract interface." It's a specific implementation of the scheduler interface for `bin/simple-scheduler`. We should keep the word "simple" next to "scheduler" in some way so that it's clearly a compound noun and not an adjective.

Reply:
Oh of course sorry I somehow didn't make that connection from your comment, I now see that you've worded this to match other scheduler classes "Implementation of ...". How about "Implementation of the abstract Scheduler class for the bundled simple-scheduler."? Then add extra lines to the docstring that point to bin/simple-scheduler and give a brief explanation of its purpose.

Reply:
Done in 1fb7b42a84f2bfe2c1b830741f7cba37e6c54ce1. Thanks for the feedback!